utils/fields.py

```diff
@@ -262,10 +262,23 @@ def generate_fake_data(self, faker, overrides=None, require_all=False):
         if self.discriminator and d_value:
             overrides[self.discriminator] = d_value
 
+        # Address fields are related so we want to have city, state, zip values that are valid instead of independent
+        # random ones.  This relies on the swagger definition using the name "Address" and the properties being named
+        # the same across APIs.
+        virtual_city_state_zip = None
+        if self.is_address_object():
```

Comment:
Unfortunately once the API is parsed we have no idea if say a field like `pickupAddress` referenced the `Address` definition in the `$ref` property.  I am testing for the required fields instead so that if a new address field gets added we don't need an allowlist of all possible property names like `destinationAddress`, ...